internals/cli/cmd_notices_test.go

```diff
@@ -99,12 +104,12 @@ func (s *PebbleSuite) TestNoticesFilters(c *C) {
 	})
 
 	rest, err := cli.Parser(cli.Client()).ParseArgs([]string{
-		"notices", "--abs-time", "--type", "custom", "--key", "a.b/c", "--type", "warning"})
+		"notices", "--abs-time", "--select", "all", "--uid", "1000", "--type", "custom", "--key", "a.b/c", "--type", "warning"})
```

Comment:
It seems to me we shouldn't be testing this combination, as it would be an API error, right? (Using both `select=all` and a user ID at once.) It doesn't need to be a CLI error, but I just don't think we should pretend this works in a test.

Reply:
This would be an API error, but this is the only function where arguments are tested, so I included both here. I suppose we could add another test function, so there is one for `--uid` and one for `--select`, but at that point, we may as well enforce in the cli that the arguments can't be used together.

Also, the usage statement doesn't mention anything about not using them together. Do you think it should say so, and I should add the additional test function and possibly the check/error in the cli if they're used together?

Reply:
Yeah, we should probably add something terse in the usage about not using them together (on one of them?). And I'd prefer if this were two tests. I don't think we should check/error in the CLI though.